WooCommerce/Classes/ViewRelated/CardPresentPayments/CardPresentPaymentsModalViewController.xib

```diff
@@ -120,8 +116,8 @@
                                                     </subviews>
                                                 </stackView>
                                                 <button opaque="NO" contentMode="scaleToFill" verticalHuggingPriority="1000" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="maO-19-PuQ">
-                                                    <rect key="frame" x="0.0" y="101" width="248" height="24"/>
-                                                    <state key="normal" title="Button"/>
+                                                    <rect key="frame" x="0.0" y="85" width="248" height="30"/>
+                                                    <state key="normal" title="Back to Order"/>
```

Comment:
nit: same as above about the placeholder text!

Reply:
Done 3c3f9276aeb67398da8f30134d8f81a491cf547e